README.md

```diff
@@ -1,3 +1,75 @@
 # Procheck
 
-A project convention checker
+A project convention checker by checking file existence and word existence in a file, using a user-defined configuration. This allows users standardize their many projects using a simple, centralized configurations.
+
+# Example Usage
+
+```
+- name: @hyperjumptech/procheck
+  uses: hyperjumptech/procheck@v1.0.0
+  with:
+    repository: '{YOUR_USERNAME}/{YOUR_REPOSITORY_NAME}'
+    configPath: '{PATH_TO_PROCHECK_JSON}'
+    githubToken: ${{ secrets.GITHUB_TOKEN }}
+```
+
+Set your secrets here: `https://github.com/USERNAME/REPO/settings/secrets`.
```

Comment:
Please add: 

Get your GITHUB_TOKEN by creating a Personal Access Token from https://github.com/settings/tokens. Please select the `repo` scope for the token.

Reply:
It should be fixed in the newest commit